build-support/docker/centos6/Dockerfile

```diff
@@ -6,16 +6,28 @@ FROM centos:6
 
 # Install python 2.7.13, (more) modern gcc, and a JDK.
 RUN yum -y update
+# TODO: figure out why this needs to be installed first for /usr/bin/scl to work!
 RUN yum install -y centos-release-scl
-RUN yum install -y \
-  devtoolset-7-gcc \
-  devtoolset-7-gcc-c++ \
+RUN yum install -y devtoolset-7-gcc{,-c++} \
   git \
   java-1.8.0-openjdk-devel \
   python27 \
-  libffi \
-  libffi-devel \
-  openssl-devel
+  libffi{,-devel} \
+  openssl-devel \
+  sqlite-devel
+
```

Comment:
Consider pyenv for the downloading / building. This might allow uniform pythons under OSX and Linux like we have for pex:
https://github.com/pantsbuild/pex/blob/812a34fb69cb16143aa780c32b40c9aa2f0c479f/.travis.yml#L1-L73

See also: https://github.com/pantsbuild/pants/pull/6981#issuecomment-453761428

Reply:
Will look into pyenv now.

Reply:
+1 for the pyenv suggestion. It's super nifty.

Reply:
It [appears to be working](https://travis-ci.org/pantsbuild/pants/builds/478875995?utm_source=github_status&utm_medium=notification) as of 3b3b77e?